src/components/Search.scss

```diff
@@ -0,0 +1,23 @@
+@import '../styles/colors.scss';
+
+/**
+*Sets the div container for the input (search) element.
+**/
+.search-container{
+  width: 80%;
+  margin: 0 auto;
+}
+
```

Comment:
We probably dont need that much whitespace between rules - one line should be fine